src/main/resources/full.dcignore

```diff
@@ -0,0 +1,1547 @@
+# Write glob rules for ignored files.
```

Comment:
@ArtsiomCh thanks for the pr, I recalled chatting about getting only Java, JS and Pythong ignores initially. From what I understand I see all ignores here. Could you please shed some light on this?

Reply:
Found only Java and Python "default" .gitignore here https://github.com/github/gitignore
Looks like for JS for each particular framework - specific .gitignore is used.
Also if we create our own "default" for IntelliJ plugin .dcignore it will be different from the one currently used in VSCE and that might confuse users.
Let me know what you think: shell we stay with current default .dcignore (same as VSCE) or create our own and if latter then which specific JS .gitignore should we use?

Reply:
@deepcodeg

Reply:
so i'm leaving default .dcignore as is. That seems to be most reasonable solution.

Reply:
Didn't get any messages from github before the merge, strange. Let's see how it goes with the default one and we could change it later if needed.